server/src/main/java/org/opensearch/common/throttle/AdmissionControlSetting.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.throttle;
+
+import java.util.Map;
+
+/**
+ * Settings for a {@link AdmissionController}
+ *
+ * @opensearch.internal
+ */
+class AdmissionControlSetting {
```

Comment:
We should allow each admission controller to bring their own settings, rather than enforcing this fixed format.